### PR TITLE
refactor into report to close #235

### LINKED ISSUE #235: Analyst exercise may need to be updated
Need to read through it to determine if a report should be added.

### PR DESCRIPTION
**Pull request by bschmoker**
I had amended the commit, dcb77ff is the canonical one (should work properly on a merge)

### CODE REVIEW

File: getting-started/analyst-exercise/output.xml [hunk 1]
```diff
@@ -30,15 +30,42 @@
...
+            <report:Header>
+                <report:Title> FireEye Ajax/Saffron report</report:Title>
+                <report:Description>Indicators and background on a computer intrusion</report:Description>
+            <report:Information_Source>
+                <stixCommon:Time>
+                    <cyboxCommon:Produced_Time>2014-05-15T00:00:00</cyboxCommon:Produced_Time>
+                </stixCommon:Time>
+            </report:Information_Source>    
+            
+            <report:Package_Intent xsi:type="stixVocabs:PackageIntentVocab-1.0">Threat Report</report:Package_Intent>
```
Comment (johnwunder) [inline getting-started/analyst-exercise/output.xml]: This should use `ReportIntentVocab-1.0`, not `PackageIntentVocab-1.0` (they are identical). I'll fix and merge, just wanted you to know for the future.

File: getting-started/analyst-exercise/output.xml [hunk 2]
```diff
@@ -30,15 +30,42 @@
...
-        <stix:Description>Indicators for FireEye report on Saffron</stix:Description>
-        <stix:Information_Source>
-            <stixCommon:Time>
-                <cyboxCommon:Produced_Time>2014-05-15T00:00:00</cyboxCommon:Produced_Time>
-            </stixCommon:Time>
-        </stix:Information_Source>
-    </stix:STIX_Header>
+    
+    <stix:Reports>
+        <stix:Report id="fireeye-ajax" xsi:type="report:ReportType">
```
Comment (johnwunder) [inline getting-started/analyst-exercise/output.xml]: This should use our best practice for IDs: `[prefix]:[objtype]-[guid]`. I can fix and merge, just wanted you to know for the future.

**Comment by johnwunder** [comment]
xml is not really valid at all either

**Comment by johnwunder** [comment]
this should merge to stix-1.2, not master

**Comment by bschmoker** [comment]
@johnwunder If I'm reading 9315c52e4a9795d952baea55973b666c69042520 correctly, making it valid was:
add `xmlns:report="http://stix.mitre.org/Report-1"` and update to version = 1.2 ` ?

**Comment by johnwunder** [comment]
IIRC it was:

- `xmlns:report=....`
- Re-order the `Reports` element to be at the end
- Re-name `Package_Intent` => `Intent`
- I also meant to switch the vocab from `PackageIntentVocab-1.0` to `ReportIntentVocab-1.0` but it looks like I forgot. It'll validate either way but only the latter is correct. Also I may have a merge error, looking into it now.

I've been using the 2.2.0 branch of STIX validator to validate things.

**Pull request closed without merging**